docs/staticAnalysis.md

```diff
@@ -103,6 +121,21 @@ To set it up, [install node.js](https://nodejs.org/en/download/) if necessary (v
 npm install -g eslint@{version}
 ```
 
+#### Installing ESLint from within IntelliJ
+
+1. Ensure the [NodeJS Plugin](https://plugins.jetbrains.com/idea/plugin/6098-nodejs) is installed.
+1. Refer to [this guide](https://www.jetbrains.com/help/idea/2016.3/using-javascript-code-quality-tools.html#ESLint) to install ESLint (version 3.11.1).
+1. Follow the same steps outlined in the guide above to install `eslint-plugin-json` (version 1.2.0).
```

Comment:
Don't mention the version numbers explicitly. Ask them to refer to `src/main/resources/package.json`.